DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/ReviewConfirmationRequests.cshtml

```diff
@@ -59,7 +59,7 @@
       <tr role="row" class="nhsuk-table__row">
         <td role="cell" class="nhsuk-table__cell nhsuk-u-font-size-16">
           <span class="nhsuk-table-responsive__heading">Comptenecy </span>
-          @DisplayStringHelper.RemoveMarkup(competency.Description)

```

Comment:
Is this string helper used elsewhere in the solution? If not, we should remove it.

Reply:
Not at the moment and it can be deleted. But there are two places where this helper could be useful in the future. 1) When validating competency `Description` is empty (_OverviewTable.cshtml:53) as Description allows Html entities like `&npsp;` and tags without content. 2) As a replacement for `SignpostingHelper.DisplayText()` which also removes markup and spaces. For (1) this code would handle null and empty `<p class="@(DisplayStringHelper.RemoveMarkup(competency.Description).Trim() != "" ? "nhsuk-u-font-weight-bold" : "") nhsuk-u-margin-bottom-0">` in the view and `@using DigitalLearningSolutions.Web.Helpers` is needed.

Reply:
Thanks Daniel. That makes sense, I will add a tech debt ticket to replace and remove SignpostingHelper.DisplayText() with DisplayStringHelper.RemoveMarkup, then.